client/site-profiler/components/health-section/index.tsx

```diff
@@ -0,0 +1,93 @@
+import page from '@automattic/calypso-router';
+import { useTranslate } from 'i18n-calypso';
+import { useMemo } from 'react';
+import { HostingProvider } from 'calypso/data/site-profiler/types';
+import { useUrlPerformanceMetricsQuery } from 'calypso/data/site-profiler/use-url-performance-metrics-query';
+import { MetricsInsight } from 'calypso/site-profiler/components/metrics-insight';
+import { InsightContent } from 'calypso/site-profiler/components/metrics-insight/insight-content';
+import { InsightHeader } from 'calypso/site-profiler/components/metrics-insight/insight-header';
+import { MetricsSection } from 'calypso/site-profiler/components/metrics-section';
+import { getTitleTranslateOptions } from 'calypso/site-profiler/utils/get-title-translate-options';
+
+interface HealthSectionProps {
+	url?: string;
+	hash?: string;
+	hostingProvider?: HostingProvider;
+	healthMetricsRef: React.RefObject< HTMLObjectElement >;
+	setIsGetReportFormOpen?: ( isOpen: boolean ) => void;
+}
+
+const OVERALL_SCORE_THRESHOLD = 0.8;
+
+export const HealthSection: React.FC< HealthSectionProps > = ( props ) => {
+	const translate = useTranslate();
+	const { url, hash, hostingProvider, healthMetricsRef, setIsGetReportFormOpen } = props;
+	const { data } = useUrlPerformanceMetricsQuery( url, hash );
+	const { truncated, diagnostic: healthData = {} } = data?.audits.health ?? {};
+
+	const isWPcom = hostingProvider?.slug?.toLowerCase() === 'automattic';
+
+	const healthScore = data?.overall_score ?? 0;
+	const isHealthGood = healthScore >= OVERALL_SCORE_THRESHOLD;
+
+	const title = useMemo( () => {
+		if ( ! isHealthGood && ! isWPcom ) {
+			return translate(
+				"Your site's health scores need attention to prevent {{poor}}low performance{{/poor}}. Improve with WordPress.com's tools for better performance.”",
```

Comment:
Remove the extra apostrophe here:

```suggestion
				"Your site's health scores need attention to prevent {{poor}}low performance{{/poor}}. Improve with WordPress.com's tools for better performance.",
```

Reply:
Nice catch https://github.com/Automattic/wp-calypso/pull/91312/commits/b63ee8781dd44f9e50cb18aeac3786746a660603